### PR TITLE
Jormungandr: add more check to a test

### PR DESCRIPTION
**Pull request by antoine-de**
the test was not very clear and difficult to understand, so add more
checks

### CODE REVIEW

**Comment by pbougue** [comment]
PR title is not clear to me. Should it be "_add_ more check to a test"?

**Comment by antoine-de** [comment]
you're right :+1:

**Comment by antoine-de** [comment]
thanks @nlehuby for the review!

File: source/jormungandr/tests/overlapping_routing_tests.py [hunk 1]
```diff
@@ -69,10 +69,20 @@ def test_journeys(self):
...
         """
-        response = self.query("/v1/{q}".format(q=journey_basic_query), display=False)
+        response = self.query("/v1/{q}".format(q=journey_basic_query))
 
         is_valid_journey_response(response, self.tester, journey_basic_query)
 
+        assert len(response['feed_publishers']) == 1
+        assert response['feed_publishers'][0]['name'] == u'routing api data'
+
+        # with the initial response we cannot specifilay check taht the empty_routing_test region
```
Comment (pbougue) [inline source/jormungandr/tests/overlapping_routing_tests.py]: typo : _specifically_ and _that_

File: source/jormungandr/tests/overlapping_routing_tests.py [hunk 2]
```diff
@@ -69,10 +69,20 @@ def test_journeys(self):
...
         """
-        response = self.query("/v1/{q}".format(q=journey_basic_query), display=False)
+        response = self.query("/v1/{q}".format(q=journey_basic_query))
 
         is_valid_journey_response(response, self.tester, journey_basic_query)
 
+        assert len(response['feed_publishers']) == 1
+        assert response['feed_publishers'][0]['name'] == u'routing api data'
+
+        # with the initial response we cannot specificaly check that the empty_routing_test region
```
Comment (pbougue) [inline source/jormungandr/tests/overlapping_routing_tests.py]: typo : specificalLy :angel:

**Pull request merged**